data/sql/updates/pending_db_world/A_Rough_Ride.sql

```diff
@@ -0,0 +1,90 @@
+("283080", 64, 5992.961, 4699.5537, -99.39478, NULL, 'Captive Crocolisk'),
+("283080", 65, 6015.1387, 4743.7524, -97.55353, NULL, 'Captive Crocolisk'),
+("283080", 66, 6035.1826, 4788.787, -94.706436, NULL, 'Captive Crocolisk'),
+("283080", 67, 6064.9507, 4827.5024, -94.57277, NULL, 'Captive Crocolisk'),
+("283080", 68, 6065.5703, 4870.553, -94.4718, NULL, 'Captive Crocolisk'),
+("283080", 69, 6096.6123, 4885.741, -94.4861, NULL, 'Captive Crocolisk'),
+("283080", 70, 6120.3867, 4902.048, -95.10951, NULL, 'Captive Crocolisk'),
+("283080", 71, 6139.616, 4913.3486, -94.939, NULL, 'Captive Crocolisk');
+
+-- Update SmartAI
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 28308;
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = 28308);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `event_param6`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(28308, 0, 1, 0, 54, 0, 100, 512, 0, 0, 0, 0, 0, 0, 80, 2830800, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Captive Crocolisk - On Just Summoned - Run Script'),
+(28308, 0, 2, 3, 58, 0, 100, 512, 71, 0, 0, 0, 0, 0, 41, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Captive Crocolisk - On Path 0 Finished - Despawn Instant'),
+(28308, 0, 3, 0, 61, 0, 100, 512, 0, 0, 0, 0, 0, 0, 15, 12536, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Captive Crocolisk - On Path 0 Finished - Quest Credit \'A Rough Ride\''),
+(28308, 0, 4, 0, 40, 0, 100, 512, 17, 0, 0, 0, 0, 0, 54, 4000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Captive Crocolisk - On Point 17 of Path Any Reached - Pause Waypoint'),
+(28308, 0, 5, 0, 40, 0, 100, 512, 43, 0, 0, 0, 0, 0, 54, 4000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Captive Crocolisk - On Point 43 of Path Any Reached - Pause Waypoint');
+
+-- Update Action List
+DELETE FROM `smart_scripts` WHERE (`source_type` = 9 AND `entryorguid` = 2830800);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `event_param6`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(2830800, 9, 1, 0, 0, 0, 100, 512, 0, 0, 0, 0, 0, 0, 53, 1, 283080, 0, 12536, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Captive Crocolisk - Actionlist - Start Waypoint Path 283080');
```

Comment:
Value and Comment should be 2830800  not 283080

<img width="737" height="69" alt="image" src="https://github.com/user-attachments/assets/a5b1ccae-2079-486d-bef0-fbe328d4bbb8" />